posthog/api/plugin.py

```diff
@@ -437,11 +438,12 @@ def _activity_page_response(
 class PluginConfigSerializer(serializers.ModelSerializer):
     config = serializers.SerializerMethodField()
     plugin_info = serializers.SerializerMethodField()
+    delivery_rate_1d = serializers.SerializerMethodField()
```

Comment:
can we make this customizable, e.g. at the project level - some people don't check posthog daily, so for them seeing a delivery_rate_7d would make more sense

Reply:
We can always refactor these queries as we learn more so let's add complexity as user feedback demands not beforehand. Adding more toggles generally leads to more complexity for both us and the end user and makes things harder to change.